tests/hdl/test_ir.py

```diff
@@ -951,8 +951,8 @@ def test_assign_names_to_fragments_collide_with_signal(self):
 
 class ElaboratableTestCase(ToriiTestSuiteCase):
 	def test_formal_on_nonformal_elaboratable(self):
+		from torii import Module
```

Comment:
Per your own repo rule, this should have the import spaced out to line up

Reply:
You are correct, these imports got glossed over due to being so far down